.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         go-version-file: 'go.mod'
 
     - name: Test
+      env:
+        GRABIT_ARTIFACTORY_TOKEN: team2grabit
```

Comment:
I don't think that's a good idea. You want to control this in code via https://pkg.go.dev/testing#B.Setenv. This way you can also test what happens when it's *not* set.

Reply:
I thought this way it may work, but okay, I will use your suggestion right away.